### ISSUE TITLE
Use btrfs on amigo PCs

### ISSUE THREAD

**Issue by Timple** [tue-robotics/tue_robocup#362]
This filesystem allows for easy and fast snapshotting.

We do quite good version control using git. But restoring the computer to the exact state of yesterday still seems tricky.

Choosing the btrfs filesystem would enable this.

**Comment by Rayman** [comment]
Good idea, never thought about that

**Comment by MatthijsBurgh** [comment]
@Timple fully implemented? Any documentation?

**Comment by Timple** [comment]
No, has to be selected on ubuntu install. So waiting until we need a clean install of a PC ;-)
